### ISSUE TITLE
Don't put AppImages into archives

### ISSUE THREAD

**Issue by TheAssassin** [electron-userland/electron-builder-binaries#5]
I've noticed that your AppImages are provided as `.7z` archives. You should not put AppImages into archives, as this prevents AppImages' "download, make executable, run" workflow. Also, type 2 AppImages are already compressed using either gzip or xz (LZMA), so they don't really need to be compressed any more. Rather change your build workflow to call appimagetool with `--comp=xz`, if you really need the additional compression. Although I think that for this tiny application, even gzip should suffice.

Furthermore, putting them into archives prevents the usage of AppImageUpdate, which I'd recommend you to provide to your users for easier updates. This is fairly easy to implement especially for GitHub releases, and if you were interested, the AppImage folks could assist.

**Comment by develar** [comment]
I am sorry for confusion.

> electron-builder downloads required tools files on demand (e.g. to code sign windows application, to make AppX).

This repository contains various tools requited for electron-builder to build something.

`appimage-9.0.1` contains darwin appimagetool, linux ia32 and x64 extracted appimagetool, and linux packages for Electron apps. Even more — we don't use appimagetool as provided AppImage because it doesn't work on Docker.

> change your build workflow to call appimagetool with --comp=xz, if you really need the additional compression

Unrelated to this issue — we decided to not use `-comp=xz`, because 
1) it doesn't work as expected (e.g. expected size for sample app  should be 35 MB, but we got 51 (1.5 MB difference compared to gzip) — because of compression level and dict size, I guess.
2) it can slowdown start-up / increase delta updates.
